internal/super/setup.go

```diff
@@ -77,134 +77,163 @@ func create(
-						ContractName: contract.Name,
-					},
-				})
-			}
-		}
+	params := config.InitConfigParameters{
+		ServiceKeySigAlgo:  "ECDSA_P256",
+		ServiceKeyHashAlgo: "SHA3_256",
+		Reset:              false,
+		Global:             false,
+		TargetDirectory:    tempDir,
+	}
+	state, err = config.InitializeConfiguration(params, logger, state.ReaderWriter())
+	if err != nil {
+		return "", fmt.Errorf("failed to initialize configuration: %w", err)
+	}
 
-		// Add the selected core contracts as dependencies
-		installer, err := dependencymanager.NewDependencyInstaller(logger, state, false, targetDir, dependencymanager.Flags{})
-		if err != nil {
-			logger.Error(fmt.Sprintf("Error: %v", err))
-			return nil, err
-		}
+	// Generate standard cadence files
+	// cadence/contracts/DefaultContract.cdc
+	// cadence/scripts/DefaultScript.cdc
+	// cadence/transactions/DefaultTransaction.cdc
+	// cadence/tests/DefaultContract_test.cdc
 
-		if err := installer.AddMany(dependencies); err != nil {
-			logger.Error(fmt.Sprintf("Error: %v", err))
-			return nil, err
-		}
+	directoryPath := filepath.Join(tempDir, "cadence")
 
-		err = state.Save(filepath.Join(targetDir, "flow.json"))
-		if err != nil {
-			return nil, err
+	_, err = generateNew([]string{"DefaultContract"}, "contract", directoryPath, logger, state)
+	if err != nil {
+		return "", err
+	}
+
+	_, err = generateNew([]string{"DefaultScript"}, "script", directoryPath, logger, state)
+	if err != nil {
+		return "", err
+	}
+
+	_, err = generateNew([]string{"DefaultTransaction"}, "transaction", directoryPath, logger, state)
+	if err != nil {
+		return "", err
+	}
+
+	// Prompt to ask which core contracts should be installed
+	sc := systemcontracts.SystemContractsForChain(flowGo.Mainnet)
+	promptMessage := "Select the core contracts you'd like to install:"
+
+	contractNames := make([]string, 0)
+
+	for _, contract := range sc.All() {
+		contractNames = append(contractNames, contract.Name)
+	}
+
+	selectedContractNames, err := prompt.RunSelectOptions(contractNames, promptMessage)
+	if err != nil {
+		fmt.Printf("Error running dependency selection: %v\n", err)
+		os.Exit(1)
+	}
+
+	var dependencies []flowkitConfig.Dependency
+
+	// Loop standard contracts and add them to the dependencies if selected
+	for _, contract := range sc.All() {
+		if slices.Contains(selectedContractNames, contract.Name) {
+			dependencies = append(dependencies, flowkitConfig.Dependency{
+				Name: contract.Name,
+				Source: flowkitConfig.Source{
+					NetworkName:  flowkitConfig.MainnetNetwork.Name,
+					Address:      flowsdk.HexToAddress(contract.Address.String()),
+					ContractName: contract.Name,
+				},
+			})
 		}
+	}
 
+	// Add the selected core contracts as dependencies
+	installer, err := dependencymanager.NewDependencyInstaller(logger, state, false, tempDir, dependencymanager.Flags{})
+	if err != nil {
+		logger.Error(fmt.Sprintf("Error: %v", err))
+		return "", err
```

Comment:
```suggestion
		return "", err
```

this should be handled by returning an error from the command